Source/WebCore/svg/SVGLength.h

```diff
@@ -48,7 +48,8 @@ class SVGLength : public SVGValueProperty<SVGLengthValue> {
         SVG_LENGTHTYPE_MM           = static_cast<unsigned>(SVGLengthType::Millimeters),
         SVG_LENGTHTYPE_IN           = static_cast<unsigned>(SVGLengthType::Inches),
         SVG_LENGTHTYPE_PT           = static_cast<unsigned>(SVGLengthType::Points),
-        SVG_LENGTHTYPE_PC           = static_cast<unsigned>(SVGLengthType::Picas)
+        SVG_LENGTHTYPE_PC           = static_cast<unsigned>(SVGLengthType::Picas),
+        SVG_LENGTHTYPE_REMS           = static_cast<unsigned>(SVGLengthType::Unknown)
```

Comment:
I don't think, it is needed. I didn't added it for `lh` and `ch` length unit and WPT progressed without issue.

Reply:
removed